config/locales/pace.en.yml

```diff
@@ -60,13 +61,13 @@ en:
         - id: what-information-will-i-have-to-provide
           heading: What information will I have to provide?
           content_html: <p>You’ll need to explain your personal circumstances in some detail – although this can be uncomfortable we need to make sure our advice is right for you in the long term. There may be options on how you provide some of your financial information including through Open banking.</p>
-        - id: what-is-open-banking
-          heading: What is Open Banking?
-          content_html: <p>Depending on who your bank is, you may have the option to share your some of your financial information with our advisers digitally. They’ll then be able to see what regularly comes into and goes out of your accounts – as long as you give them your permission. This is a tried and tested process which is completely secure.</p>
-                        <p>It also means you and our advisers can be certain that we’ve got accurate and up to date information which can help you get to the right solution quickly.</p>
-        - id: is-open-banking-confidential
-          heading: Is Open Banking confidential?
-          content_html: <p>Yes. Full details of how we make sure your information stays secure can be found <a href="/en/moneyadvisernetwork/privacy" target="_blank">here</a></p>
+        # - id: what-is-open-banking
+        #   heading: What is Open Banking?
+        #   content_html: <p>Depending on who your bank is, you may have the option to share your some of your financial information with our advisers digitally. They’ll then be able to see what regularly comes into and goes out of your accounts – as long as you give them your permission. This is a tried and tested process which is completely secure.</p>
+        #                 <p>It also means you and our advisers can be certain that we’ve got accurate and up to date information which can help you get to the right solution quickly.</p>
+        # - id: is-open-banking-confidential
+        #   heading: Is Open Banking confidential?
+        #   content_html: <p>Yes. Full details of how we make sure your information stays secure can be found <a href="/en/moneyadvisernetwork/privacy" target="_blank">here</a></p>
```

Comment:
Should we leave all these comments?

Reply:
I've now removed these comments. Left them there in case someone's mind changed in the interim ;)